README.org

```diff
@@ -265,10 +268,68 @@ releases. Use the ~fill~ command instead.
 report that it filled in more than one block. This is expected, and will occur
 when orphans/forks are present at that height.
 
+*** migrate
+
+~migrate~ allows you to migrate the database schema to the latest version and exit.
+This can be useful for separating the migration step from running the ETL and/or HTTP service.
+
+#+begin_example
+  > chainweb-data migrate --dbuser=joe --dbname=chainweb-data
+#+end_example
+
+
 *** check-schema
 
 ~check-schema~ is used to perform a check of the ORM definitions against the DB schema.
 
 #+begin_example
   > chainweb-data check-schema --service-host=foo.chainweb.com --dbuser=joe --dbname=chainweb-data
 #+end_example
+
+** Specializing the Database Schema
```

Comment:
Just for my understanding, is this not Markdown? I see the usage of `~code~` instead of <code>`</code>. The extension is also different, `md` vs `org`. Probably a Haskell convention?

Reply:
This README is written as an Emacs Org mode document. I wouldn't say it's a Haskell convention, it's probably more of an Emacs person thing :) (which I'm not, this document predates me).